apps/drain-safe/src/components/Balances.tsx

```diff
@@ -104,6 +105,7 @@ function Balances({
       sortedByHeaderId={orderBy}
       sortDirection={order}
       onHeaderClick={handleHeaderClick}
+      onRowClick={handleExclusion}
```

Comment:
TableRow is not an interactive element, and we shouldn't add any event listeners to it. This is inaccessible.

https://stackoverflow.com/questions/64776155/make-table-row-or-cell-be-read-by-a-table-as-clickable

Reply:
I'm just using the safe component library @mikheevm and is the requirement in this US

Reply:
Perhaps is this is really important should be addressed in the library but a row click event i think is no that strange. Suggestions in stackoverflow i think overcomplicate this

Reply:
This is inaccessible. People that use screen readers to navigate through the web page won't be able to use this.
I suggest you read this: https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility.

> Suggestions in stackoverflow i think overcomplicate this

What's the overcomplication in making a button?

Reply:
I created a [ticket](https://github.com/gnosis/safe-react-components/issues/169) in the component library

Reply:
I don't mean adding a button is complicated. I'm not considering it because the requirements in this ticket are to add the click in the row level. I'm referring to the other solutions in the link to stackoverflow you provided. 

Anyway, we can reformulate the requirement of course @dasanra

Reply:
@yagopv We can consider this for later. Was an appreciation from designers, but as I wrote in the ticket not a hard requirement if there is a good tech reason.

I think this was thought as for mobile devices, where you can click a full row to select or unselect more easily than having to pick a small checkbox

Reply:
@yagopv @mikheevm I found after some investigation that Material UI already provides a component to solve this issue
https://mui.com/components/tables/#data-table

This table is not using table html components and is much more convenient for modern table uses. We can consider if it may be worth to migrate to this kind of table instead traditional structure, that is not that friendly to accessibility.

But anyway to close this ticket I would just remove this requirement and add all the other improvements

Reply:
I checked the implementation out of curiosity: https://github.com/mui-org/material-ui-x/blob/e45cf31723303ccc20cf21cccb6ef1e975364bed/packages/grid/_modules_/grid/components/GridRow.tsx#L225 🤯 

Making such a component accessible is a lot of work; developing a component library is no joke

Reply:
ok removed the row event!!

Reply:
If we want truly accesible tables i think we need a complete solution like the mui one @dasanra commented because if you want to navigate this table using a screen reader I think is is completely imposible as roles and indexes are missing

Reply:
We don't have to "want" accessibility. It should be included by default.